recipes/libgpuarray/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set version = "0.6.0" %}
+
+package:
+  name: libgpuarray
+  version: "0.6.0"
+
+source:
+  fn: v{{ version }}.tar.gz
```

Comment:
Could change this to `libgpuarray-{{ version }}.tar.gz` to keep the name distinct for local builds.